.env.template

```diff
@@ -0,0 +1,4 @@
+DATABASE_URL=postgres://postgres:mysecretpassword@localhost:5432/mydatabase?sslmode=disable
+GOOGLE_CLIENT_ID=google-client-id
+JWT_SECRET=jwt-secret
+SERVER_URL=server-url
```

Comment:
Always good to document public settings, such as environment variables, that the users will interact with 👍

```suggestion
# DB connection URL
DATABASE_URL=postgres://postgres:mysecretpassword@localhost:5432/mydatabase?sslmode=disable
# Google OAuth credentials
# 	See: https://support.google.com/cloud/answer/6158849?hl=en
GOOGLE_CLIENT_ID=google-client-id
GOOGLE_CLIENT_SECRET=google-client-secret
# JWT private key
# 	Generate using: `openssl rand -hex 32`
JWT_SECRET=jwt-secret
# API server URL
#	The external URL, usually used for the contruction of links
SERVER_URL=server-url
```

Reply:
Got you. Link this to #23 .